src/zzz_od/application/coffee/coffee_app.py

```
@@ -123,18 +123,49 @@ def move_and_interact(self) -> OperationRoundResult:
-        return self.round_by_find_area(self.last_screenshot, '咖啡店', '点单',
-                                       success_wait=1, retry_wait=1)
+        if self.config.transport_point == CoffeeTransportPoint.POINT_1.value.value:  #六分街-咖啡店
+            return self.round_by_find_area(self.last_screenshot, '咖啡店', '点单',
+                                           success_wait=1, retry_wait=1)
+
+        return self.round_success(status='对话点单', wait=1)  # 新版咖啡店
+
+    @node_from(from_name='等待咖啡店加载', status='对话点单')
+    @operation_node(name='对话选咖啡', node_max_retry_times=10)
+    def dialog_choose_coffee(self) -> OperationRoundResult:
+        """推进对话点单分支，直到出现咖啡选项或回到大世界"""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)